dome9/resource_dome9_cloudaccount_aws.go

```diff
@@ -306,20 +303,33 @@ func resourceCloudAccountAWSUpdate(d *schema.ResourceData, meta interface{}) err
 }
 
 func expandCloudAccountAWSRequest(d *schema.ResourceData) aws.CloudAccountRequest {
+	credentials := expandCloudAccountAWSCredentials(d)
+	vendor := "aws"
```

Comment:
why is the vendor not taken from the resource? where is the vendor validation against the credentials?